_posts/en/newsletters/2019-08-14-newsletter.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Bitcoin Optech Newsletter #59'
+permalink: /en/newsletters/2019/08/14/
+name: 2019-08-14-newsletter
+slug: 2019-08-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter briefly describes two discussions on the
+Bitcoin-Dev mailing list, one about Bitcoin vaults and one about
+reducing the size of public keys used in taproot.  Also included are our
+regular sections about bech32 sending support and notable changes in
+popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Optech [San Francisco][workshop-sf] (Sep 24) and [New York
+  City][workshop-nyc] (Sep 27) workshops:**  member companies should [send us an
+  email][optech email] to reserve spots for your engineers. Planned topics
+  include presentations and coding demos involving schnorr, MuSig, taproot, and tapscript.
```

Comment:
nit: despite https://github.com/bitcoinops/bitcoinops.github.io/issues/171 it is strikes me as somewhat odd that MuSig is uppercased WRT to the others

Reply:
MuSig was dropped in an edit, but it's a good point.